.gitignore

```diff
@@ -396,6 +396,7 @@ model.index\+
 # local environment settings used by e.g. Visual Studio Code
 /.env
 /doc/source/local_config.json
+/.venv*
```

Comment:
I'm adding this as .venv* is a common name pattern used to create local virtual environment.